package.json

```diff
@@ -10,7 +10,8 @@
     "start:staging": "API_URL=https://lego-staging.abakus.no/api/v1 BASE_URL=https://lego-staging.abakus.no WS_URL=wss://ws-staging.abakus.no CAPTCHA_KEY=1x00000000000000000000AA ENVIRONMENT=local_staging yarn start",
     "ssr:staging": "API_URL=https://lego-staging.abakus.no/api/v1 BASE_URL=https://lego-staging.abakus.no WS_URL=wss://ws-staging.abakus.no CAPTCHA_KEY=1x00000000000000000000AA yarn ssr",
     "ssr": "echo \"Remember that SSR does not have hot reloading, build with \"yarn build\"\" && node ./dist/server.js",
-    "build": "concurrently \"yarn run build:server\" \"yarn run build:client\"",
+    "build": "yarn build:packages && concurrently \"yarn run build:server\" \"yarn run build:client\"",
```

Comment:
Does Bricks have to be built before both server and client?

Reply:
Yeah, because it is imported as a node module, we need it before we start building the webapp.